.github/workflows/build-and-publish.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       run: mvn clean verify sonar:sonar --batch-mode -Dsonar.projectKey=SHOGun -Dsonar.login="${{ secrets.SONARQUBE_TOKEN }}" -Preporting
 
     - name: Refresh SonarQube
-      if: steps.semantic.outputs.new_release_published == 'true'
+      if: ${{ github.actor != 'dependabot[bot]' && steps.semantic.outputs.new_release_published == 'true'
```

Comment:
I think `}}` is missing here

or `${{` isn't needed

Reply:
you are absolutely right thank you!